Libraries/Components/ScrollResponder.js

```diff
@@ -112,6 +114,27 @@ type State = {
 };
 type Event = Object;
 
+if (Platform.OS === 'android') {
+  var AndroidTextInput = requireNativeComponent('AndroidTextInput', null);
```

Comment:
This doesn't seem safe -- depending on the order things are loaded, we could cache a version of the native component with no view config.

Maybe we can just set nativeComponentName on the return of requireNativeComponent so that we can check `instance._currentElement.type.nativeComponentName` below? Not sure how @gaearon feels about that.